assets/styles/components/_media.scss

```diff
@@ -78,6 +78,10 @@
       }
     }
   }
+  
+  .image-form {
```

Comment:
Did you try this on mobile? Setting a width in pixels is often causing problems on mobile...

Reply:
It works fine on mine

![Screenshot_20240825-163636](https://github.com/user-attachments/assets/6eae9818-ccf3-451d-852c-c9980e8990bf)

Reply:
It works fine on mine

![Screenshot_20240825-163636](https://github.com/user-attachments/assets/8b25d7c2-1c57-4690-8471-f8bf90fc1f47)

Reply:
Depending on the DPI and display.. It could maybe result into issues indeed. You might want to use percentage (%) or vh

Reply:
Fixed is another way, see latest commit. The image-form class is now obsolete. Should I remove it from 
><div class="image-form">

Reply:
I actually don't like the change. To test this you can hit f12 on your desktop when you're on the site and (if you're on Firefox) click this little phone icon: 
![Bildschirmfoto vom 2024-08-27 12-25-14](https://github.com/user-attachments/assets/8bade073-bfc6-4f04-b936-091d25cc9f5f)

With that you can test different mobile devices.

I actually don't understand which problem you were trying to solve when changing the width setting. If I apply a `width: 100%` to the `image-form` class it looks fine to me, scales very well with all display sizes...

Reply:
As to why; with no changes, it looked like this after the buttons removed: (impression)

![image](https://github.com/user-attachments/assets/223f25c8-8fae-49ba-ae94-f54cdaed02ec)

which is solved by removing the  width: 100%; from .image-input

Thanks for the tip on the little camera, I always wondered why FF had no such feature, had to resort to chrome every time.

I have applied your suggestion width: 100% to image-form

Reply:
Ah ok, then I misread which width you removed. The difference is only visible if you make the window very small, but I think it is fine this way :+1: